external-docs/docs/guides/shape-data.md

```diff
@@ -34,78 +34,9 @@ Both data shapes expect that data is structured as per the following example:
-
-The simplest [qube-config.json](./configuration/qube-config.md) we can define for this data set is:
-
-```json
-{
-    "$schema": "http://purl.org/csv-cubed/qube-config/v1",
-    "title": "'Arthur's Bakes' stores in UK cities from 2020 to 2022",
-    "description": "The number of 'Arthurs' Bakes' stores in cities across the UK between 2020 and 2022.",
-    "creator": "https://www.gov.uk/government/organisations/hm-revenue-customs",
-    "publisher": "https://www.gov.uk/government/organisations/hm-revenue-customs",
-    "columns": {
-        "Year": {
-            "type": "dimension"
-        },
-        "Location": {
-            "type": "dimension"
-        },
-        "Value": {
-            "type": "observations"
-        },
-        "Status": {
-            "type": "attribute"
-        },
-        "Measure": {
-            "type": "measures"
-        },
-        "Unit": {
-            "type": "units"
-        }
-    }
-}
-```
-
-It is possible to use the [configuration by convention approach](./configuration/convention.md) to generate a valid [standard shape](#standard-shape) cube without defining a [qube-config.json](./configuration/qube-config.md) at all. Just ensure that your columns use the [conventional column names](./configuration/convention.md#conventional-column-names) appropriate to their type.
-
-### Multiple Measures
-
-One of the benefits of the standard shape is that it is relatively straightforward to add new measure types and unit types; all that you have to do is add additional rows to your data set with the appropriate units and measures present.
-
-We can extend our example data set so that it now includes revenue values for the given year by adding rows to the table:
-
-| Year | Location | Value |      Status |                    Measure |                   Unit |
-|:-----|:---------|------:|------------:|---------------------------:|-----------------------:|
-| 2022 | London   |    35 | Provisional | Number of 'Arthur's Bakes' |                  Count |
-| 2022 | London   |    25 | Provisional |                    Revenue | GBP Sterling, Millions |
-| 2021 | Cardiff  |    26 |       Final | Number of 'Arthur's Bakes' |                  Count |
-| 2021 | Cardiff  |    18 |       Final |                    Revenue | GBP Sterling, Millions |
-
-The same data could be represented in the [equivalent pivoted shape](#multiple-pivoted-measures):
-
-| Year | Location | Number of 'Arthur's Bakes' | Number of Stores Status | Revenue (GBP Sterling, Millions) | Revenue Status |
-|:-----|:---------|---------------------------:|:------------------------|---------------------------------:|:---------------|
-| 2022 | London   |                         35 | Provisional             |                               25 | Provisional    |
-| 2021 | Cardiff  |                         26 | Final                   |                               18 | Final          |
-
 ## Converting to the standard shape
```

Comment:
What do you think about moving the conversion sections into the child document pages?

Reply:
Done - I've also moved the shape pages further up in the navbar, as they felt a bit buried